CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+## Fides Contribution Guidelines
+
+The Fides project, which includes Fideslang, Fidesops, and Fidesctl, adheres to the following [Code of Conduct](https://ethyca.github.io/fidesops/community/code_of_conduct/).
```

Comment:
I know we're just directing them to the code of conduct here, but maybe we could rephrase this to be a little more welcoming -

Reply:
Thank you for this! I love being more welcoming but tell me if this update is crossing a line :)

Reply:
I really like this improvement! if I am considering contributing a bugfix to a project, I look at the README first, usually CONTRIBUTING.md second, and sometimes check to see if a code of conduct or similar exists.  An initial perception of friendliness make me more willing to go through the effort.